src/tsickle_compiler_host.ts

```diff
@@ -0,0 +1,199 @@
+import * as ts from 'typescript';
+
+import {ModulesManifest} from './modules_manifest';
+import * as tsickle from './tsickle';
```

Comment:
It is, but everything seems to be working fine - should I be concerned that nothing's breaking?